### PR TITLE
Adding Troubleshooting Tip

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4302

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Teams-add-in-for-Outlook.md [hunk 1]
```diff
@@ -171,6 +171,9 @@ If the add-in still does not show, use the following steps to verify the registr
 
 If your administrator has configured Microsoft Exchange to [control access to Exchange Web Server (EWS)](https://docs.microsoft.com/exchange/client-developer/exchange-web-services/how-to-control-access-to-ews-in-exchange), a delegate won't be able to schedule a Teams meeting on behalf of the boss. The solution for this configuration is under development and will be released in the future. As a workaround, your administrator can add the following string to the EWS Allow List: "*SchedulingService*". 
 
+### Incorrect Meeting Formatting
+
+If you have issues with proper formatting when setting up Microsoft Teams meetings in Outlook and your organization uses the VBA For Office Addin, you may need to enable that addin to ensure that the meeting will format properly. Refer to [View, manage, and install add-ins in Office programs](https://support.microsoft.com/office/view-manage-and-install-add-ins-in-office-programs-16278816-1948-4028-91e5-76dca5380f8d) for the process that best applies to your version of Outlook.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:176]:
```suggestion
If you have issues with proper formatting when setting up Microsoft Teams meetings in Outlook and your organization uses the Microsoft Visual Basic for Applications for Microsoft Office add-in, you may need to enable that add-in to ensure that the meeting will format properly. Refer to [View, manage, and install add-ins in Office programs](https://support.microsoft.com/office/view-manage-and-install-add-ins-in-office-programs-16278816-1948-4028-91e5-76dca5380f8d) for the process that best applies to your version of Outlook.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by TokyoScarab** [comment]
Just following up on this. Thanks!

**Comment by TokyoScarab** [comment]
@LanaChin Any news?

**Comment by LanaChin** [comment]
@TokyoScarab - See the comment I added in the GitHub issue. We need more details around the nature of the issue. For example, what aspect of the invitations aren't displayed correctly?

**Comment by LanaChin** [comment]
Closing for now. Replied to the submitter to open a new GH issue and include details around the nature of the problem they're experiencing, if this is still an issue for them.  Per the product team, we need additional details before making any documentation updates.

**Pull request closed without merging**